internal/namespaces/instance/v1/testdata/test-list-server-simple.stdout.golden

```diff
@@ -1,12 +1,12 @@
-ed30e9df-4360-43f8-b02b-dfe23d793a83  cli-srv-zealous-cartwright    running   fr-par-1  51.158.126.164  10.64.142.157  Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  f974feac-abae-4365-b988-8ec7d1cec10d  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group  booting kernel  x86_64
-aa4fedfc-4937-4ce9-8e05-b5f8b2b5678c  scw-dreamy-saha               running   fr-par-1  51.158.64.183   10.65.26.161   Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  f974feac-abae-4365-b988-8ec7d1cec10d  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group  booting kernel  x86_64
+ID                                    NAME                          STATE     ZONE  PUBLIC IP       PRIVATE IP     IMAGE NAME            TAGS  MODIFICATION DATE  CREATION DATE    IMAGE ID                              PROTECTED  VOLUMES  SECURITY GROUP ID                     SECURITY GROUP NAME     STATE DETAIL    ARCH
+6ee01533-3fb9-4f20-8160-a7ad7967c6d4  cli-srv-festive-antonelli     archived        163.172.133.14                 Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  f974feac-abae-4365-b988-8ec7d1cec10d  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group                  x86_64
+a5beed5f-7406-4574-b204-b49c5bd51aad  cli-srv-nice-mirzakhani       archived        51.158.72.126                  Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  d4067cdc-dc9d-4810-8a26-0dae51d7df42  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group                  x86_64
+d4ed923f-943a-40fc-89ca-d5bdc1b7be88  cli-srv-reverent-hodgkin      archived        51.158.99.155                  Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  d4067cdc-dc9d-4810-8a26-0dae51d7df42  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group                  x86_64
+93eb4197-46ae-4c81-b6cf-1222a861edd2  cli-srv-upbeat-burnell        archived        51.158.119.154                 Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  d4067cdc-dc9d-4810-8a26-0dae51d7df42  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group                  x86_64
+9424a299-8cf3-4125-9471-0d10d4df0412  cli-srv-hopeful-jennings      archived        163.172.177.30                 Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  d4067cdc-dc9d-4810-8a26-0dae51d7df42  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group                  x86_64
+e34e7ca4-b5aa-4522-9cba-d16122f041f6  cli-srv-hardcore-einstein     running         51.158.120.130  10.68.26.121   Ubuntu Bionic Beaver  []    few seconds ago    few seconds ago  f974feac-abae-4365-b988-8ec7d1cec10d  false      1        49822f6c-1d32-465c-a017-720691d159b2  Default security group  booting kernel  x86_64
```

Comment:
should zone really disappear ? 

this looks like a regression

Reply:
The issue comes from the cassette because the field zone was not returned by the API until recently.